assembly/assembly-ide-war/pom.xml

```diff
@@ -347,6 +345,19 @@
                 <artifactId>exec-maven-plugin</artifactId>
                 <executions>
                     <execution>
+                        <id>gwt-xml</id>
+                        <phase>generate-sources</phase>
+                        <goals>
+                            <goal>java</goal>
+                        </goals>
+                        <configuration>
+                            <mainClass>org.eclipse.che.util.CompilingGwtXmlGenerator</mainClass>
```

Comment:
example of custom packaging type http://blog.sonatype.com/2009/08/create-a-customized-build-process-in-maven/